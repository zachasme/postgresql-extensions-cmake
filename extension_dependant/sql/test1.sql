\set epsilon 0.0000000000001
\set lyon POINT(4.8422, 45.7597)
\set paris POINT(2.3508, 48.8567)

SELECT dependant('8a283082a677fff');

SELECT h3_great_circle_distance(:lyon, :paris, 'km') - 392.21715988417765 < :epsilon;
