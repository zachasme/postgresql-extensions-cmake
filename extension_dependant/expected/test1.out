SELECT dependant(42);
 dependant 
-----------
        42
(1 row)

